CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 2.8)
+
+include(GNUInstallDirs)
+
+find_package(LLVM REQUIRED CONFIG)
+message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+
+add_definitions(${LLVM_DEFINITIONS})
+include_directories(${LLVM_INCLUDE_DIRS})
+link_directories(${LLVM_LIBRARY_DIRS})
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -fno-rtti")
```

Comment:
I really do not like `CMAKE_CXX_FLAGS` being modified directly because it globally effects everything. I much prefer using `target_compile_options()` and setting on a per target basis. For initial implementation this is probably fine.